modules/globebrowsing/tile/asynctilereader.h

```diff
@@ -26,36 +26,36 @@
 #define __OPENSPACE_MODULE_GLOBEBROWSING___ASYNC_TILE_READER___H__
 
 #include <modules/globebrowsing/other/concurrentjobmanager.h>
-
 #include <modules/globebrowsing/tile/tileindex.h>
 
 #include <unordered_map>
 
 namespace openspace {
 namespace globebrowsing {
     
-class RawTile;
-class TileDataset;
+struct RawTile;
+class RawTileDataReader;
 
 class AsyncTileDataProvider {
 public:
-    AsyncTileDataProvider(std::shared_ptr<TileDataset> textureDataProvider, 
+    AsyncTileDataProvider(std::shared_ptr<RawTileDataReader> textureDataProvider,
         std::shared_ptr<ThreadPool> pool);
 
     bool enqueueTileIO(const TileIndex& tileIndex);        
     std::vector<std::shared_ptr<RawTile>> getRawTiles();
```

Comment:
The RawTileDataReader can not be const because it is reading data and sould be able to reset. getRawTiles can not be const bacause AsyncTileDataProvider saves a list of all recently read RawTiles which gets cleared when calling getRawTiles.